README.md

```diff
@@ -12,16 +12,16 @@ line.
 
 On one computer the tool generates a one-time code for us:
 
-	$ cat hello.txt
-	hello, world
-	$ ww send hello.txt
-	8-enlist-decadence
+    $ cat hello.txt
```

Comment:
don't like my tabs? :-) (it's ok)

Reply:
I like tabs, but seems my vscode settings did something

Reply:
ha that's fine, we can stick with spaces.